txbridge/src/main/java/org/jboss/jbossts/txbridge/outbound/BridgeXAResource.java

```diff
@@ -87,10 +88,9 @@ public BridgeXAResource(Uid externalTxId, BridgeWrapper bridgeWrapper)
      */
     private void writeObject(ObjectOutputStream out) throws IOException
     {
-        txbridgeLogger.logger.trace("BridgeXAResource.writeObject() for tx id="+externalTxId);
```

Comment:
There are too many places where logging does not check the level first so no I am not asking you to go and change them all. My comment was so that we don't forget to do it sometime in the future.

As to including the transaction in the trace output you may as well if you are making other changes anyway.